packages/wallets/exodus/src/index.ts

```diff
@@ -1,16 +1,37 @@
-import { WalletAdapterType } from '@reef-knot/core-react';
+import { WalletAdapterType } from '@reef-knot/types';
+import { Ethereum } from '@wagmi/core';
+import { InjectedConnector } from 'wagmi/connectors/injected';
 import WalletIcon from './icons/exodus.svg';
 
+declare global {
+  interface Window {
+    // @ts-expect-error Subsequent property declarations must have the same type
+    ethereum?: Ethereum & { isExodus?: boolean };
+    exodus?: { ethereum?: Ethereum & { isExodus?: boolean } };
+  }
+}
+
 export const Exodus: WalletAdapterType = () => ({
   walletName: 'Exodus',
   walletId: 'exodus',
   icons: {
     light: WalletIcon,
     dark: WalletIcon,
   },
-  detector: () => !!window.ethereum?.isExodus,
+  detector: () =>
+    typeof window !== 'undefined'
+      ? !!window.exodus?.ethereum?.isExodus || !!window.ethereum?.isExodus
```

Comment:
you really need to check `exodus.ethereum.isExodus`? is it not enough there is `window.exodus`?